### ISSUE TITLE
content of generated file is empty.

### ISSUE THREAD

**Issue by BekzodYusupov** [grpc/grpc-kotlin#401]
I tried to implement **grpc_**(android/kst/kotlin)_ in my project. I copied gradle code from sample you provided 
https://github.com/GoogleCloudPlatform/kotlin-samples/tree/main/run/grpc-hello-world-gradle

but for some reason it cannot generate content of 1 file(**HelloworldKt.kt**). I have helloworld.protoc _(src/main/java/proto)_ 

![image](https://github.com/grpc/grpc-kotlin/assets/101949099/10fb76de-c822-4015-8673-5e68592378d9)

github: https://github.com/BekzodYusupov/grpc

_Originally posted by @BekzodYusupov in https://github.com/grpc/grpc-kotlin/issues/320#issuecomment-1687548827_

**Comment by jamesward** [comment]
One thing that might fix it is to add the `java` builtin:
```
            it.builtins {
                create("kotlin")
                create("java")
            }
```

Let me know if that does the trick.

**Comment by BekzodYusupov** [comment]
yes, now it generated files. thanks a lot :) for fast response @jamesward 
I also upgraded protobuf version to the latest which resolved duplicate java and kotlin error: from 3.20.1 to 3.24.1
